src/AgentHeader.php

```diff
@@ -87,11 +90,23 @@ public static function buildAgentHeader($headerInfo)
             : Version::getApiCoreVersion();
         $metricsHeaders['gax'] = $apiCoreVersion;
 
+        // Context on library type identification (between gRPC+REST and REST-only):
+        // This uses the gRPC extension's version if 'grpcVersion' is not set, so we
+        // cannot use the presence of 'grpcVersion' to determine whether or not a library
+        // is gRPC+REST or REST-only. However, we cannot use the extension's presence
+        // either, since some clients may have the extension installed but opt to use a
+        // REST-only library (e.g. GCE).
+        // TODO: Should we stop sending empty gRPC headers?
```

Comment:
Can we file an associated issue linking to this PR to help us follow up?

Reply:
Done.